Whiskey/Tasks/Invoke-WhiskeyDotNetPublish.ps1

```diff
@@ -0,0 +1,81 @@
+
+function Invoke-WhiskeyDotNetPublish
+{
+    [CmdletBinding()]
+    [Whiskey.Task("DotNetPublish")]
+    [Whiskey.RequiresTool('DotNet','DotNetPath',VersionParameterName='SdkVersion')]
+    param(
+        [Parameter(Mandatory=$true)]
+        [object]
+        $TaskContext,
+
+        [Parameter(Mandatory=$true)]
+        [hashtable]
+        $TaskParameter
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
+    $dotnetExe = $TaskParameter['DotNetPath']
+
+    $projectPaths = ''
+    if ($TaskParameter['Path'])
+    {
+        $projectPaths = $TaskParameter['Path'] | Resolve-WhiskeyTaskPath -TaskContext $TaskContext -PropertyName 'Path'
+    }
+
+    $verbosity = $TaskParameter['Verbosity']
+    if (-not $verbosity -and $TaskContext.ByBuildServer)
+    {
+        $verbosity = 'detailed'
+    }
+
+
+    $outputDirectory = $TaskParameter['OutputDirectory']
+    if ($outputDirectory -and ([IO.Path]::IsPathRooted($outputDirectory)))

```

Comment:
What happens if you pass an absolute path to this? Does it handle it? Can we just pass it as-is? Do we need to validate?

Reply:
Absolute path works fine. I was just following our design pattern where we generally don't allow absolute paths in Whiskey task properties.

Reply:
Since we're passing this to `dotnet.exe`, I don't think we should do any validation. Let `dotnet.exe` worry about validating it.